renderer/src/components/Saturn.tsx

```diff
@@ -1,10 +1,10 @@
 import { useEffect } from 'react'
-import { getFilAddress, startSaturnNode } from '../lib/station-config'
+import { getDestinationWalletAddress, startSaturnNode } from '../lib/station-config'
 
 const Saturn = () => {
   useEffect(() => {
     (async () => {
-      if (await getFilAddress()) {
+      if (await getDestinationWalletAddress()) {
         startSaturnNode()
```

Comment:
saturn should always start, since it only needs the `StationWalletAddress`, not the destination address

Reply:
[034e18f](https://github.com/filecoin-station/filecoin-station/pull/333/commits/034e18ffbf401148a787b69de44d570f67fd13e0)